content/How_To/animation/animation_method.md

```diff
@@ -31,7 +31,7 @@ const myAnim = new BABYLON.Animation(name, property, frames per second, property
 
 -   _loop mode_ - _number optional_, This can be set using the following Parameters
 
-    BABYLON.Animation.ANIMATIONLOOPMODE*CYCLE - Restart the animation from initial value  
+    BABYLON.Animation.ANIMATIONLOOPMODE_CYCLE - Restart the animation from initial value  
     BABYLON.Animation.ANIMATIONLOOPMODE_CONSTANT - Pause the animation at the final value  
     BABYLON.Animation.ANIMATIONLOOPMODE_RELATIVE - Repeat the animation incrementing using key value gradients. In this way, for example, a \_clip* showing a character's legs in a walking motion can be looped to show the character progressing across the scene.

```

Comment:
Also in this line, not sure if `\_clip*` is intentional.

Reply:
Probably not 😁